app/views/users/show.html.erb

```diff
@@ -1,9 +1,17 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Users Details</title>
-  </head>
-  <body>
-    <h1>Details for a given user</h1>
-  </body>
-</html>
+<div class="flex flex-col w-full">
+  <div id="userInfos">
+        <%= render partial: 'shared/user', locals: {user: @user} %>
+
+        <div id="bio" class="border-4 border-black mt-4 md:w-1/2 mx-auto mb-5">
+          <h2>Bio</h2>
+          <p><%= @user.bio %></p>
+        </div>
+  </div>
+
+  <% @user.posts.each do |post| %>
```

Comment:
- [ ] The posts here should be rendered using the `three_recent_posts` method in the User model, not all user posts